projects/igniteui-angular/src/lib/directives/mask/mask.directive.ts

```diff
@@ -105,271 +92,156 @@ export class IgxMaskDirective implements OnInit, ControlValueAccessor {
     @Output()
     public onValueChange = new EventEmitter<IMaskEventArgs>();
 
-    /**
-     *@hidden
-     */
-    private get value() {
+    /** @hidden @internal; */
```

Comment:
```suggestion
    /** @hidden @internal */
```
Also, probably no surprise here - what's with making this public? Guessing related to future PR's, but I'd rather see it change in that PR so I know what the use for it is.